xml/System.Data.Common/DbDataReader.xml

```diff
@@ -724,7 +724,6 @@ private static void GetDataTypes(String connectionString) {
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.InvalidCastException">The specified cast is not valid.</exception>
```

Comment:
Don't expect this exception from a method that doesn't retrieve data from the database.